src/store/wallet/index.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+export default {
+  namespaced: true,
+  actions: {
+    async walletBinding({commit}, data) {
+      const baseUrl = process.env.VUE_APP_DEV_DEGENICS_BACKEND_URL
+      const result = await axios.post(`${baseUrl}/substrate/wallet-binding`, data , {
+      headers: {
+        "debio-api-key" : "9NG;zEyyy@{{;>5~s3;dNw1A[I,@K6"
```

Comment:
NEVER, hard code API Keys, please undo this commit and force push your changes to remove this from the file history @ciniiia